etc/shadowing.yaml.dist

```diff
@@ -0,0 +1,36 @@
+# This file is an example file for using DOMjudge as a shadow system.
+# The `import/import-event-feed` command uses the file etc/shadowing.yaml to
+# read configuration options for shadowing another system following the
+# Contest API specification (https://clics.ecs.baylor.edu/index.php?title=Contest_API).
+
+# THe root keys are identifiers used to pass to `import/import-event-feed`
```

Comment:
nit: s/THe/The/

Reply:
Also "used to pass to `import/import-event-feed`" --> "used to pass to `import/import-event-feed`, e.g. use `import/import-event-feed exampleA` to import the first contest.

"